dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/datavalueset/CsvDataValueSetReader.java

```diff
@@ -51,11 +51,17 @@ final class CsvDataValueSetReader implements DataValueSetReader, DataValueEntry
     @Override
     public DataValueSet readHeader()
     {
+        DataValueSet set = new DataValueSet();
         if ( importOptions == null || importOptions.isFirstRowIsHeader() )
         {
             readNext(); // Ignore the first row: assume header row
+            String dataSetId = getString( 11 );
+            if ( dataSetId != null && !dataSetId.isEmpty() )
```

Comment:
Minor, could use apache StringUtils.isNotEmpty for slightly more compact and readable code.

Reply:
done